src/Tree.h

```diff
@@ -105,29 +105,24 @@ Tree<T>::Tree( T * pParent )
 , mActive( false )
 , mUserActiveState( false )
 {
-    if( pParent )
-    {
+    if (pParent) {
         pParent->addChild((T*)(this));
-    }
-    else
+    } else
```

Comment:
Too late here but I thought we were aiming to have (manually inserted 'cause clag-format doesn't do it) `{`...`}` around the single line cases like the `else` here... :see_no_evil: